.travis.yml

```diff
@@ -6,16 +6,16 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
   - hhvm
-  - nightly
```

Comment:
Let's leave `nightly` in here. It may be redundant with 7.3 for some period of time, but going forward we'll always want to test against the latest build.